elyra/pipeline/processor.py

```diff
@@ -67,7 +67,7 @@ async def process(self, pipeline):
             raise RuntimeError('Could not find pipeline processor for [{}]'.format(pipeline.runtime))
 
         processor.root_dir = self.root_dir
-        res = await asyncio.get_event_loop().run_in_executor(None, processor.process, pipeline)
+        res = await processor.process(pipeline)
```

Comment:
After looking at the other changes, this change doesn't appear to be necessary and I suspect the previous async behavior may be broken since nothing in the (now) async methods is awaiting to allow for the processing to continue elsewhere.